src/states/game.gd

```diff
@@ -23,7 +23,7 @@ var target_id := 0
 # A list of the indices into target_transforms for the last group of spawned targets.
 var last_spawned_target_group: Array[int] = []
 # A list of all the possible spawn locations within the current level.
-var spawn_points := []
+var spawn_points: Array[Node] = []
```

Comment:
Below (line 227) I see you said `var spawn_point: SpawnPoint`. Is it possible to use that as a type here?
```gdscript
var spawn_points: Array[SpawnPoint] = []
```

Reply:
Nope, the editor complains if I do that. It gets upset when assigning the value of get_nodes_in_group() to that variable.

Reply:
Does it let you do `get_nodes_in_group() as Array[SpawnPoint]`?

Reply:
Nope, I tried that as well

Reply:
cringe ahh ahh

ok we'll leave it for now